docs/docusaurus/docs/cloud/try_gx_cloud.md

```diff
@@ -21,6 +21,12 @@ If you've tested GX Cloud features and functionality and discovered it's a great
 
 - You have a [Docker instance](https://docs.docker.com/get-docker/).
 
+## Exploratory deployment pattern
+
+To try GX Cloud, you will run the GX Agent locally via Docker, connect the GX Agent to your target Snowflake data stores, and use the GX Cloud web UI to define your Data Assets and validate your data. This architecture is suited for an exploratory use case and differs from the recommended production setup, in which the GX Agent [runs in your organization's deployment environment](./about_gx#gx-cloud-architecture).
```

Comment:
```suggestion
To try GX Cloud features and functionality, you run the GX Agent locally with Docker, connect the GX Agent to your target Snowflake Data Assets, and use the GX Cloud web UI to create Expectations and validate your data. This architecture is suited for testy use cases and differs from the recommended deployment setup, in which the GX Agent [runs in your organization's deployment environment](./about_gx#gx-cloud-architecture).
```

From the [Google developer documentation style guide](https://developers.google.com/style/word-list#letter-v) (our primary style resource): 

![image](https://github.com/great-expectations/great_expectations/assets/104205257/2378d6e2-202d-4b4a-91b5-5d4d33ff1724)

This text is great, by the way.